node/pkg/driver/node_utils.go

```diff
@@ -177,6 +179,23 @@ func (n NodeUtils) StageInfoFileIsExist(filePath string) bool {
 	}
 	return true
 }
+
+func (n NodeUtils) DevicesAreNvme(sysDevices []string) bool {
+	args := []string{"list"}
+	out, err := n.Executer.ExecuteWithTimeout(TimeOutMultipathdCmd, nvmeCmd, args)
+	if err != nil {
+		return false
+	}
+	devices := string(out)
+	for _, deviceName := range sysDevices {
+		if strings.Contains(devices, deviceName) {
```

Comment:
which of the "lists" is potentially longer?
I think we should check inside the one that is expected to be shorter

Reply:
sysDevices is way shorter

Reply:
then I think it is more suitable for `Contains`

Reply:
then again, the first sys device should be contained in nvme devices, so I'm not sure why we want to potentially go over all the volume sys devices

Reply:
should we skip the rescan if **any** of them is nvme, or **all**?

Reply:
it should find it on the first check, (if it checks nvmeDevices Contains sysDevice). no reason for going over all  the sysDevices

Reply:
so do we need a for loop?
and if not, should the function even require a **list** of sys devices?

Reply:
maybe if we have some sort of hybrid-connectivity configuration per volume, we should go over all sys devices before deciding to entirely skip the rescan?

Reply:
I'm not sure which of the scenarios is even possible, hope you're right, resolving